pallets/ocex/src/lib.rs

```diff
@@ -437,8 +385,9 @@ pub mod pallet {
 		pub fn add_proxy_account(origin: OriginFor<T>, proxy: T::AccountId) -> DispatchResult {
 			let main_account = ensure_signed(origin)?;
 			ensure!(Self::orderbook_operational_state(), Error::<T>::ExchangeNotOperational);
-			// TODO: Avoid duplicate Proxy accounts
 			ensure!(<Accounts<T>>::contains_key(&main_account), Error::<T>::MainAccountNotFound);
+			// Avoid duplicate Proxy accounts
+			ensure!(!<Proxies<T>>::contains_key(&proxy), Error::<T>::ProxyAlreadyRegistered);
```

Comment:
After adding this assertion - benchmark ([this](https://github.com/Polkadex-Substrate/Polkadex/blob/28e7a135e932e0fecdd14508fddc1f30b1e45e58/pallets/ocex/src/benchmarking.rs#L92-L109)) fails on calling `add_proxy_account` extrinsic with error:
> Error: Input("Benchmark pallet_ocex_lmp::add_proxy_account failed: ProxyAlreadyRegistered")

Please update extrinsic as well.

Reply:
Resolved